data/src/main/java/com/globant/data/mapper/HouseMapper.kt

```diff
@@ -0,0 +1,16 @@
+package com.globant.data.mapper
+
+import com.globant.data.service.response.HouseResponse
+import com.globant.domain.entity.House
+
+class HouseMapper : BaseMapper<HouseResponse, House> {
+    override fun transform(type: HouseResponse): House = type.run {
+        House(
+            this._id,
+            this.name
+        )
+    }
+
+    fun transformListOfHouseId(housesResponse: List<HouseResponse>): List<House> =
```

Comment:
recheck de name pls, I think should be transformListOfHouse

Reply:
Refactored!